addons/stock/wizard/stock_transfer_details.py

```diff
@@ -91,16 +92,17 @@ def do_detailed_transfer(self):
                     'owner_id': prod.owner_id.id,
                 }
                 if prod.packop_id:
-                    prod.packop_id.write(pack_datas)
+                    prod.packop_id.with_context(self._context, no_recompute=True).write(pack_datas)
```

Comment:
This changes indeed everything.  I wonder if we could not even do without the recompute in all cases.